docs/reports/requirements/2_dfd-desc.md

```diff
@@ -1,25 +1,20 @@
 # Data Flow Diagrams
 
+These diagrams are generalizations about how our system will interact with data from the user, Kukarella Web App, and our Animation Add-On. The level 0 DFD highlights a high overview of the system and the level 1 DFD breaks down the system in more detail. The system we are building is working alongside an existing system. Items highlighted in red represent the parts of the system that are already in place and items highlighted in blue are parts of the system that are being added by our new Animation Add-On.
 
-## Level 0 DFD 
-
-![](dfd_0.png)
+## Level 0 
 
+The level 0 data flow diagrams give a high level view of how our system will be interacting with the user and the client’s existing system. 
 
+![](dfd_0.png)
 
+The user will first access Kukurella’s text-to-speech front end and type in a text input. The user will also select a voice and an avatar they would like to use for their text-speech animation. From the front-end the text and other user selections will be sent over to our Animation Add-On. The Animator will then query an avatar image, avatar coordinates, and mouth animation assets from a database. The Animator will produce an animation of an avatar reading the user’s text in the user selected voice. This animation will be sent to Kukeralla's web application and then will be sent to the user.  
```

Comment:
If you'd like to avoid saying "text-to-speech" a lot, I did abbreviate it in the project description to TTS, so you can do that, too!